packages/office-ui-fabric-react/package.json

```diff
@@ -71,6 +71,7 @@
     "sinon": "^4.1.3"
   },
   "dependencies": {
+    "@fluentui/react-focus": "^0.1.0",
```

Comment:
Can we not do this dependency in production? The docs literally say: no production code should use a v0.

Let's stage this. Leave the old code. Publish. Then in a separate PR do this and validate that nothing breaks.

Reply:
Done!